doc/state_estimation_nodes.rst

```diff
@@ -174,6 +174,10 @@ If *true*, the state estimation node will publish the transform from the frame s
 ^^^^^^^^^^^^^^^^^^^^^
 If *true*, the state estimation node will publish the linear acceleration state. Defaults to *false*.
 
+~permit_corrected_publication
```

Comment:
Any "new in XYZ distro" marker we can do?

Reply:
Should we just make a `noetic-devel` and then change this PR to target that? Then we don't have to worry about that.

Reply:
Yeah